lib/TabsNew/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { node, string, oneOfType, arrayOf, shape } from 'prop-types';
+import cx from 'classnames';
+import { TabsBase } from './TabsBase';
+
+function Tabs({ children, className, actions, tabs }) {
+  const tabsClassName = cx('flex flex-wrap w-full relative', {
+    'overflow-y-hidden': tabs.length <= 24,
+    'overflow-y-scroll': tabs.length > 24
+  });
+
+  return (
+    <div className="relative">
+      <nav className={`flex max-h-30 ${tabsClassName} ${className}`}>
+        {children}
+      </nav>
+      {actions && <div className="absolute right-0 mr-4">{actions}</div>}
```

Comment:
💚big opinion here, I'm not a huge fan of passing elements/components as props. Instead you could have...

```
<Tabs>
    <Tabs.Nav>
        some tabs as children
    </Tabs/Nav>
    <Tabs.Actions>
        a button or something
    </Tabs.Actions>
</Tabs>

Reply:
💚 Same, but echoing what Amee said, this is totally opinion on my part - unless we sit down and decide as a team we should go one way and not the other, this is as valid as Amee's suggestion.